docs/src/main/sphinx/release/release-354.md

```diff
@@ -0,0 +1,80 @@
+# Release 354 (XXXXXXXX Mar 2021)
+
+## General
+
+* Preferred write partitioning is now triggered automatically when the number
+  of written partitions exceeds or is equal to
+  ``preferred-write-partitioning-min-number-of-partitions``. The number of written
+  partitions is estimated from stats, therefore CTAS or insert source tables needs to have
+  statistics collected. To forcefully enable preferred partitioning,
+  ``preferred-write-partitioning-min-number-of-partitions`` needs to be set to
+  ``0`` or ``1``. To fully disable preferred write partitioning, set
+  ``use-preferred-write-partitioning`` to ``false``.
+  ({issue}`6920`)
+* Improve performance of certain queries involving `LEFT`, `RIGHT` and `FULL JOIN` 
+  when one side of the join is known to produce a single row. ({issue}`7090`)
+* Fix failure when subqueries contain `EXCEPT` or `INTERSECT`. ({issue}`7342`)
+* Fix incorrect results when multiple similar subqueries contain `UNION` clauses that differ 
+  only in the `ALL` vs `DISTINCT` qualifier. ({issue}`7345`)
+
+## Security
+
+* Fix OAuth 2.0 repeated authentication, after previous token expires. ({issue}`7172`)
+* Support OAuth 2.0 authentication in Trino CLI. ({issue}`7054`)
+
+## ClickHouse connector
+
+* Use correct case for name of the schema in `CREATE SCHEMA`. ({issue}`7239`)
+
+## Elasticsearch connector
+
+* Fix failure when reading single-valued fields for array types. ({issue}`7012`)
+
+## Hive connector
+
+* Fix possible S3 connection leaks when using AVRO table format ({issue}`7178`)
+* Fix query failure when columns of a CSV table are declared as a type other than `varchar` in Glue metastore. Columns 
```

Comment:
`varchar` -> `varchar` (`string`) ?

Reply:
Do columns get declared as `string` or as `varchar` in Glue?

Reply:
this i do not know

Reply:
I do not neither. But I would expect that without the fix it would work fine previously for both cases. Only if columns in Glue were neither `varchar` nor `string` it would fail.